assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/SavedIdeas.js

```diff
@@ -16,8 +16,110 @@
  * limitations under the License.
  */
 
-const SavedIdeas = () => {
-	return <div className="googlesitekit-idea-hub__saved-ideas" />;
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Fragment, useCallback, useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import {
+	IDEA_HUB_BUTTON_CREATE,
+	IDEA_HUB_BUTTON_UNPIN,
+	IDEA_HUB_IDEAS_PER_PAGE,
+	STORE_NAME,
+} from '../../../datastore/constants';
+import EmptyIcon from '../../../../../../svg/idea-hub-empty-saved-ideas.svg';
+import PreviewTable from '../../../../../components/PreviewTable';
+import Idea from './Idea';
+import Empty from './Empty';
+import Footer from './Footer';
+const { useSelect } = Data;
+
+const SavedIdeas = ( { WidgetReportError } ) => {
```

Comment:
👍 Actually I was testing something and forgot to remove the `active` prop in `index.js`. I'am not sure though, that we need to actually return `null` here since the content is already hidden via CSS.